File: WePeiYang/gpa/src/main/java/com/twtstudio/retrox/gpa/GpaApi.java
package com.twtstudio.retrox.gpa;



import retrofit2.http.GET;
import rx.Observable;

/**
 * Created by retrox on 2017/1/26.
 */

public interface GpaApi {
    @GET("gpa")
    Observable<MyGpaBean> getGpa();
}

File: WePeiYang/gpa2/src/main/java/xyz/rickygao/gpa2/api/GpaApi.kt
package xyz.rickygao.gpa2.api

import retrofit2.http.FieldMap
import retrofit2.http.FormUrlEncoded
import retrofit2.http.GET
import retrofit2.http.POST
import rx.Single


/**
 * Created by rickygao on 2017/11/9.
 */
interface GpaApi {
    @GET("gpa")
    fun get(): Single<Response<GpaBean>>

    @FormUrlEncoded
    @POST("gpa/evaluate")
    fun evaluate(@FieldMap params: Map<String, String>): Single<Response<String>>
}
